functions/transferCallResolve.ts

```diff
@@ -0,0 +1,66 @@
+import '@twilio-labs/serverless-runtime-types';
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+import {
+  responseWithCors,
+  bindResolve,
+  error400,
+  error500,
+  success,
+} from 'tech-matters-serverless-helpers';
+
+const TokenValidator = require('twilio-flex-token-validator').functionValidator;
+
+type EnvVars = {
+  TWILIO_WORKSPACE_SID: string;
+};
+
+export type Body = {
+  taskSid?: string;
+  reservationSid?: string;
+};
+
+async function closeTask(context: Context<EnvVars>, body: Required<Body>) {
```

Comment:
I'm using this name because in the task is "gone" from the agent panel after this, but what is really happening here is that the reservation is being completed. Should I change the name for this function?

Reply:
The function name does seem out of sync with the code inside of it, which has `closedReservation`.  So `closeReservation` seems to be more natural.